### PR TITLE
Initalizing the bundle for the builds for Openshift operator

### PR DESCRIPTION
**Pull request by ayushsatyam146**
Initalizing the bundle for the builds for Openshift operator and generating manifests using `operator-sdk generate kustomize manifests `. Adding custom Builds for openshift operator icon and adding relevant links ion the CSV.

### CODE REVIEW

**Comment by avinal** [comment]
Please add pull request description.

File: config/manifests/bases/operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: operator.v0.0.0
+  namespace: placeholder
```
Comment (sayan-biswas) [inline config/manifests/bases/operator.clusterserviceversion.yaml:8]: the namespace should be `openshift-build`

File: config/manifests/bases/operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: operator.v0.0.0
```
Comment (sayan-biswas) [inline config/manifests/bases/operator.clusterserviceversion.yaml:7]: semantic version is required.

File: config/manifests/bases/operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
```
Comment (sayan-biswas) [inline config/manifests/bases/operator.clusterserviceversion.yaml:11]: we'll need the `shipwrightbuilds.operator.shipwright.io` CRD also

File: config/manifests/bases/operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    mediatype: image/svg+xml
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
```
Comment (sayan-biswas) [inline config/manifests/bases/operator.clusterserviceversion.yaml:28]: @avinal This refers to you story.

File: config/manifests/bases/operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  maintainers:
+  - email: sabiswas@redhat.com
+    name: Sayan Biswas
+  - email: adkaplan@redhat.com
+    name: Adam Kaplan
+  maturity: alpha
+  provider:
+    name: Red Hat
+    url: https://github.com/redhat-openshift-builds
+  version: 0.0.0
```
Comment (sayan-biswas) [inline config/manifests/bases/operator.clusterserviceversion.yaml:57]: semantic version required

File: bundle/tests/scorecard/config.yaml [hunk 6]
```diff
@@ -0,0 +1,70 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
```
Comment (sayan-biswas) [inline bundle/tests/scorecard/config.yaml:1]: Not sure whether we need scorecard deployment.

**Review by sayan-biswas — CHANGES_REQUESTED** [review]

**Review by sayan-biswas — APPROVED** [review]

**Pull request merged**